main.cpp

```diff
@@ -141,27 +226,37 @@ int main()
     auto carRented = rentACar(6, 2); 
     
     //1)
-    
+    auto phoneRinged = ringThePhone(10);
+
     //2)
+    auto ledTurnedOn = turnOnServerLed(4, true);
     
     //3)
+    auto buttonPushed = pushButton(3, 5);
```

Comment:
the first function argument has the type `float`.
you're passing an `int`.  The compiler is implicitly converting this to a `float`.
when initializing floats, use the proper syntax: `3.f` or `3.0f`
- [ ] revise